lang/java/reef-applications/reef-vortex/src/main/java/org/apache/reef/vortex/evaluator/VortexWorker.java

```diff
@@ -53,15 +54,22 @@
 
   private final BlockingDeque<byte[]> pendingRequests = new LinkedBlockingDeque<>();
   private final BlockingDeque<byte[]> workerReports = new LinkedBlockingDeque<>();
+  private final ConcurrentMap<Integer, AggregateContainer> aggregates = new ConcurrentHashMap<>();
 
+  private final AggregateFunctionRepository aggregateFunctionRepository;
+  private final VortexAvroUtils vortexAvroUtils;
   private final HeartBeatTriggerManager heartBeatTriggerManager;
   private final int numOfThreads;
   private final CountDownLatch terminated = new CountDownLatch(1);
 
   @Inject
   private VortexWorker(final HeartBeatTriggerManager heartBeatTriggerManager,
+                       final AggregateFunctionRepository aggregateFunctionRepository,
```

Comment:
Isn't `AggregateFunctionRepository` for driver? Could you explain how this will be used in Workers?